docs/development/architecture/ASTS.md

```diff
@@ -4,7 +4,7 @@ title: ASTs
 sidebar_label: ASTs
 ---
 
-## Abstract Syntax Trees (AST)s
+# Abstract Syntax Trees (AST)s
```

Comment:
@Josh-Cena was this intentional? The lint complaint is correct:

```
docs/development/architecture/ASTS.md:7 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# Abstract Syntax Trees (AST)s"]
```

Reply:
Ah wait, I didn't start the site locally... Forgot to hide the front matter title